dev/io.openliberty.netty.internal.impl/src/io/openliberty/netty/internal/impl/UDPUtils.java

```diff
@@ -0,0 +1,158 @@
+        bs.applyConfiguration(config);
+        bs.group(framework.parentGroup);
+        bs.channel(NioDatagramChannel.class);
+        return bs;
+    }
+
+    /**
+     * UDP bind() helper
+     * 
+     * @param bootstrap
+     * @param inetHost
+     * @param inetPort
+     * @param retryCount
+     * @param retryDelay
+     * @param bindListener
+     * @return
+     */
+    private static ChannelFuture bindHelper(NettyFrameworkImpl framework, BootstrapExtended bootstrap, String inetHost, int inetPort, final int retryCount, final int retryDelay, ChannelFutureListener bindListener) {
+        ChannelFuture bindFuture = bootstrap.bind(inetHost, inetPort);
+        if (bindListener != null) {
+            bindFuture.addListener(bindListener);
+        }
+        final String channelName = ((UDPConfigurationImpl) bootstrap.getConfiguration()).getExternalName();
+        
+        bindFuture.addListener(future -> {
+            if (future.isSuccess()) {
+                framework.activeChannels.add(bindFuture.channel());
+
+                // set common channel attrs
+                bindFuture.channel().attr(ConfigConstants.NameKey).set(channelName);
+                bindFuture.channel().attr(ConfigConstants.HostKey).set(inetHost);
+                bindFuture.channel().attr(ConfigConstants.PortKey).set(inetPort);
+
+                // UDP CWWKO0400I listening message
+                Tr.info(tc, UDPMessageConstants.UDP_CHANNEL_STARTED, new Object[] { channelName, inetHost, String.valueOf(inetPort) });
+            } else {
+                if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+                    Tr.debug(tc, "bindHelper failed due to: " + future.cause().getMessage());
+                }
+
+                if (retryCount > 0) {
+                    if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+                        Tr.debug(tc, "attempt to bind again after a wait of " +retryDelay +"ms; " + retryCount + " attempts remaining");
+                    }
+                    // recurse until we either complete successfully or run out of retries;
+                    try {
+                        Thread.sleep(retryDelay);
+                    } catch (InterruptedException x) {
+                        // do nothing but debug
+                        if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+                            Tr.debug(tc, "sleep caught InterruptedException.  will proceed.");
+                        }
+                    }
+                    bindHelper(framework, bootstrap, inetHost, inetPort, retryCount - 1, retryDelay, bindListener);
+                } else {
+                    Tr.error(tc, UDPMessageConstants.BIND_FAILURE, new Object[] { channelName, inetHost, String.valueOf(inetPort) });
+                }
+            }
+        });
+        return bindFuture;
+    }
+
+    /**
+     * UDP start helper implementation 
```

Comment:
...but this start 'helper's method name is not postfixed by Helper but the bindHelper is?
Personally, in a *Utils class, for static methods that will have that classname on the call line
I think the postfix 'Helper' does not add much except extra chars. Your choice, but best to be consistent
unless these is some reason.

Reply:
I agree with getting rid of the _Helper_ postifx